config/base-tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "outDir": "dist",
```

Comment:
is this relative to the individual app folder or the git repo - should it be `./dist` ?

Reply:
Good catch.  Fixed.